test/qpy_compat/test_qpy.py

```diff
@@ -130,6 +130,9 @@ def generate_random_circuits():
 
 def generate_string_parameters():
     """Generate a circuit from pauli tensor opflow."""
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", DeprecationWarning)
+        from qiskit.opflow import X, Y, Z
     op_circuit = (X ^ Y ^ Z).to_circuit_op().to_circuit()
     op_circuit.name = "X^Y^Z"
     return op_circuit
```

Comment:
I think this method just used opflow as a convenient way to generate a quantum circuit, so we actually don't use opflow for anything QPY-related, why don't we replace it with the new version already?
```suggestion
def generate_string_parameters():
    """Generate a circuit for the XYZ pauli string."""
    op_circuit = QuantumCircuit(3, name="X^Y^Z")
    op_circuit.z(0)
    op_circuit.y(1)
    op_circuit.x(2)
    return op_circuit
```